python/lsst/pipe/tasks/processCcdWithFakes.py

```diff
@@ -370,10 +385,13 @@ def copyCalibrationFields(self, calibCat, sourceCat):
                 missingFieldNames.append(fieldName)
         if missingFieldNames:
             raise RuntimeError(f"calibCat is missing fields {missingFieldNames} specified in "
-                               "calibrationFieldsToCopy")
+                               "fieldsToCopy")
 
-        self.calibSourceKey = calibSchemaMapper.addOutputField(afwTable.Field["Flag"]("calib_detected",
-                                                               "Source was detected as an icSource"))
+        if "calib_detected" not in calibSchemaMapper.getOutputSchema():
+            self.calibSourceKey = calibSchemaMapper.addOutputField(afwTable.Field["Flag"]("calib_detected",
+                                                                   "Source was detected as an icSource"))
+        else:
+            self.calibSourceKey = None
```

Comment:
Is this None because if 'calib_detected' is already in the calibSchemaMapper.getOutputSchema(), then it will be set as part of the match process?